### ISSUE TITLE
401 Login reuired

### ISSUE THREAD

**Issue by triadsk** [spatie/laravel-analytics#34]
Hi

this is my config (I've replaced real values with *xxxx* placeholders):
```php
    [
        'siteId' => 'ga:xxxxxxxx',

        'clientId' => 'xxxxxxxxxxxx-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.apps.googleusercontent.com',

        'serviceEmail' => 'xxxxxxxxxxxx-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx@developer.gserviceaccount.com',

        'certificatePath' => storage_path('laravel-analytics/xxxxxxxx.p12'),

        'cacheLifetime' => 0,//60 * 24 * 2,

        'realTimeCacheLifetimeInSeconds' => 5,
    ];
```
I'm making testing call from your documentation in my Controller:
```php
    public function index(LaravelAnalytics $analytics) {
        $analyticsData = $analytics->getVisitorsAndPageViews(7);
        print_r($analyticsData);
        return true;
    }
```
I'm getting *401 Login Required* error:
```
Google_Service_Exception in REST.php line 110:
Error calling GET https://www.googleapis.com/analytics/v3/data/ga?ids=&start-date=2015-09-25&end-date=2015-10-02&metrics=ga%3Avisits%2Cga%3Apageviews&dimensions=ga%3Adate: (401) Login Required
```
My credentials are definitely correct, because I've cross-checked them with *HelloAnalytics.php* example from [Google Analytics docs](https://developers.google.com/analytics/devguides/reporting/core/v3/quickstart/service-php) and I can see the data, no error is displayed.

**Comment by freekmurze** [comment]
I haven't seen this error before. As all my projects that are using this package can display ga-data perfectly. I'm guessing that there is some error in your config somewhere (or some step you missed from the [credentials-part in the readme](https://github.com/spatie/laravel-analytics#how-to-obtain-the-credentials-to-communicate-with-google-analytics).  Be sure to mention the solution if you find one.

**Comment by triadsk** [comment]
I sorted it out. The problem is, that in your service provider you are registering string 'laravelAnalytics' as a key to get your service. I was not using Fasade. I was injecting the LaravelAnalytics class directly to my controller and in that case it does not load published config.

I think changing registered string to full class name
```
$this->app->bind(LaravelAnalytics::class, function ($app) {
```
would be more intuitive, don't you think? Or is it on purpose?

**Comment by freekmurze** [comment]
Thanks for letting me know the solution. I'll treat this a bug and will fix it soon.

**Comment by freekmurze** [comment]
@triadsk this is fixed in version 1.2.2
